File: leetcode solutions/Python/excel sheet column number/excelSheetColumnNumber.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        if not s:
            return 0

        colNum = 0
        count = 0

        for i in xrange(0, len(s)):
            colNum = colNum * 26 + (ord(s[i]) - ord('A') + 1)

        return colNum

File: leetcode solutions/C++/excel sheet column number/excelSheetColumnNumber.cpp
#include <iostream>

using namespace std;
//const int N = 0;

class Solution {
public:
    int titleToNumber (string s) {
        int colNum = 0;
        for (char i : s) {
            colNum = colNum * 26 + (i - 'A' + 1);
        }
        return colNum;
    }


};

int main() {
    string str;
    Solution s;
    while (cin >> str)
        cout << s.titleToNumber(str) << endl;
    return 0;
}